PROCESS TOOK 2400 SECONDS ERROR in  benchmark_ALIBICNOGRAD.py  AND DSNAME  20 FOR C 0 AND ROW 0
PROCESS TOOK 2400 SECONDS ERROR in  benchmark_ALIBICNOGRAD.py  AND DSNAME  20 FOR C 0 AND ROW 1
